packages/core/src/animation/Animator.ts

```diff
@@ -200,11 +236,39 @@ export class Animator extends Component {
     return this._getAnimatorStateInfo(stateName, layerIndex).state;
   }
 
+  /**
+   * Get the layer by name.
+   * @param name - The layer's name.
+   */
+  findLayerByName(name: string): AnimatorControllerLayer {
+    return this._animatorController._layersMap[name];
+  }
+
+  /**
+   * Get the parameter by name.
+   * @param name - The name of the parameter
+   */
+  getParameter(name: string): AnimatorControllerParameter {
+    return this._animatorController._parametersMap[name] || null;
+  }
+
+  /**
+   * Set the value of the given parameter.
+   * @param name - The name of the parameter
+   */
+  setParameter(name: string, value: AnimatorControllerParameterValue) {
```

Comment:
if property store in sub attribute, we warp get method is OK, but warp set API is not good, It will confuse developers in understanding the data structure